yarn.lock

```diff
@@ -7968,6 +8310,26 @@ jest-cli@^24.9.0:
     realpath-native "^1.1.0"
     yargs "^13.3.0"
 
+jest-config@^23.0.1:
```

Comment:
updating to v1 of the projects plugin should remove this again https://github.com/jest-community/jest-watch-select-projects/pull/7